samples/showcase/cinema/Core_8/Cinema.Headquarters/RecordTicketSalesHandler.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.Extensions.Logging;
```

Comment:
All the lines of this file are commented out. Is this file still relevant ?

Reply:
File has been removed.